diff --git a/domain_301_redirect.info.yml b/domain_301_redirect.info.yml
index 374f719a0855fd1cc4d68148379484e017176975..e9398317dd0dc1e1f5e7cd9fe72158a149b4ce41 100644
--- a/domain_301_redirect.info.yml
+++ b/domain_301_redirect.info.yml
@@ -3,3 +3,6 @@ type: module
 description: Allows you to 301 redirect any domain to the main domain associated with a site.
 core: 8.x
 configure: domain_301_redirect.settings
+core_version_requirement: ^8 || ^9
+dependencies:
+  - drupal:path_alias
diff --git a/domain_301_redirect.services.yml b/domain_301_redirect.services.yml
index 8b6cf81702f9a2bdba4a3dd60d7f707e5ae7dad6..6b7701e9866fccc43f5e0126ae4aa3b92efd774b 100644
--- a/domain_301_redirect.services.yml
+++ b/domain_301_redirect.services.yml
@@ -1,7 +1,7 @@
 services:
   domain_301_redirect.event_subscriber:
     class: Drupal\domain_301_redirect\EventSubscriber\DomainRedirectEventSubscriber
-    arguments: ['@config.factory', '@request_stack', '@current_user', '@path.matcher', '@path.alias_manager']
+    arguments: ['@config.factory', '@request_stack', '@current_user', '@path.matcher', '@path_alias.manager']
     tags:
       - { name: event_subscriber }
 
diff --git a/src/EventSubscriber/DomainRedirectEventSubscriber.php b/src/EventSubscriber/DomainRedirectEventSubscriber.php
index 7cae64bb550f03a1a952c3a41c54f6571d3f778f..d91fab4f86a71b83c398bfd7c895a9a8888be5e2 100644
--- a/src/EventSubscriber/DomainRedirectEventSubscriber.php
+++ b/src/EventSubscriber/DomainRedirectEventSubscriber.php
@@ -6,7 +6,7 @@ use Drupal\Core\Cache\CacheableMetadata;
 use Drupal\Core\Cache\Cache;
 use Drupal\Core\Config\ConfigFactoryInterface;
 use Drupal\Core\Path\PathMatcherInterface;
-use Drupal\Core\Path\AliasManagerInterface;
+use Drupal\path_alias\AliasManagerInterface;
 use Drupal\Core\Session\AccountProxyInterface;
 use Drupal\domain_301_redirect\Domain301Redirect;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
@@ -53,7 +53,7 @@ class DomainRedirectEventSubscriber implements EventSubscriberInterface {
   /**
    * The path alias manager.
    *
-   * @var \Drupal\Core\Path\AliasManagerInterface
+   * @var \Drupal\path_alias\AliasManagerInterface
    */
   protected $pathAliasManager;
 
@@ -68,7 +68,7 @@ class DomainRedirectEventSubscriber implements EventSubscriberInterface {
    *   The current user object.
    * @param \Drupal\Core\Path\PathMatcherInterface $path_matcher
    *   Path Matcher interface.
-   * @param \Drupal\Core\Path\AliasManagerInterface $pathAliasManager
+   * @param \Drupal\path_alias\AliasManagerInterface $pathAliasManager
    *   Path alias manager interface.
    */
   public function __construct(ConfigFactoryInterface $config_factory, RequestStack $request_stack, AccountProxyInterface $user_account, PathMatcherInterface $path_matcher, AliasManagerInterface $pathAliasManager) {
