Index: .htaccess
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.htaccess b/.htaccess
--- a/.htaccess	
+++ b/.htaccess	(date 1643126257981)
@@ -88,6 +88,62 @@
   # downloaded.
   RewriteRule "/\.|^\.(?!well-known/)" - [F]
 
+  # Redirect for EHS
+  RewriteCond %{HTTP_HOST} ^(www\.)?fire\.ucla\.edu$ [NC]
+  RewriteRule ^ https://www.ehs.ucla.edu/fire [L,R=301]
+
+  # Redirects for police.ucla.edu
+  RewriteCond %{HTTP_HOST} ^(www\.)?ems\.ucla\.edu$ [NC]
+  RewriteRule ^ https://www.police.ucla.edu/ems [L,R=301]
+
+  RewriteCond %{HTTP_HOST} ^(www\.)?cso\.ucla\.edu$ [NC]
+  RewriteRule ^ https://www.police.ucla.edu/cso [L,R=301]
+
+  # Redirect for Research
+  RewriteCond %{HTTP_HOST} ^(www\.)?nagpra\.ucla\.edu$ [NC]
+  RewriteRule ^ https://www3.research.ucla.edu/nagpra [L,R=301]
+
+  # it.ucla.edu Specific Rules
+  # Redirecting all /security/ pages to https://www.ociso.ucla.edu
+    RewriteCond %{HTTP_HOST} ^(www\.)?it\.ucla\.edu$ [NC]
+    RewriteRule ^security/(.*)  https://www.ociso.ucla.edu/$1 [R=301,NC,L]
+
+  # Redirecting https://bol.ucla.edu to https://www.it.ucla.edu/bol
+    RewriteCond %{HTTP_HOST} ^(www\.)?bol\.ucla\.edu$ [NC]
+    RewriteRule ^ https://www.it.ucla.edu/bol%{REQUEST_URI} [L,R=301]
+
+  # Redirecting https://qdb.ucla.edu to https://www.it.ucla.edu/enterprise-data-warehouse
+    RewriteCond %{HTTP_HOST} ^(www\.)?qdb\.ucla\.edu$ [NC]
+    RewriteRule ^ https://www.it.ucla.edu/enterprise-data-warehouse [L,R=301]
+
+  # Redirecting http://mobile.ucla.edu
+    RewriteCond %{HTTP_HOST} ^(www\.)?mobile\.ucla\.edu$ [NC]
+    RewriteRule ^ https://ucla.service-now.com/support?id=its_browse_service&sys_id=31085bf81b5c885038cec8415b4bcb2c&table=u_service_definitions [L,R=301]
+
+  # Redirecting https://www.it.ucla.edu/applications
+    RewriteCond %{HTTP_HOST} ^(www\.)?it\.ucla\.edu$ [NC]
+    RewriteCond %{REQUEST_URI} ^/applications$ [NC,OR]
+    RewriteCond %{REQUEST_URI} ^/applications/ [NC]
+    RewriteRule ^ https://ucla.service-now.com/support?id=sc_browse_service&sys_id=85ff1f831b54041038cec8415b4bcbea [L,R=301]
+
+  # Redirecting https://www.it.ucla.edu/it-ucla/units
+    RewriteCond %{HTTP_HOST} ^(www\.)?it\.ucla\.edu$ [NC]
+    RewriteCond %{REQUEST_URI} ^/it-ucla/units$ [NC,OR]
+    RewriteCond %{REQUEST_URI} ^/it-ucla/units/ [NC]
+    RewriteRule ^ https://www.it.ucla.edu/people [L,R=301]
+
+  # Redirect for rcp
+  RewriteCond %{HTTP_HOST} ^(www\.)?rcp\.ucla\.edu$ [NC]
+  RewriteRule ^ https://www3.research.ucla.edu/rcp [L,R=301]
+
+  # Redirect for bto
+  RewriteCond %{HTTP_HOST} ^(www\.)?bto\.ucla\.edu$ [NC]
+  RewriteRule ^ https://www.it.ucla.edu/ [L,R=301]
+
+  # Redirect for ittransformation
+  RewriteCond %{HTTP_HOST} ^(www\.)?ittransformation\.ucla\.edu$ [NC]
+  RewriteRule ^ https://www.cio.ucla.edu/ [L,R=301]
+
   # If your site can be accessed both with and without the 'www.' prefix, you
   # can use one of the following settings to redirect users to your preferred
   # URL, either WITH or WITHOUT the 'www.' prefix. Choose ONLY one option:
@@ -139,6 +195,8 @@
   RewriteCond %{REQUEST_URI} !/core/[^/]*\.php$
   # Allow access to test-specific PHP files:
   RewriteCond %{REQUEST_URI} !/core/modules/system/tests/https?\.php
+  # Allow access to simplesaml paths
+  RewriteCond %{REQUEST_URI} !^/simplesaml
   # Allow access to Statistics module's custom front controller.
   # Copy and adapt this rule to directly execute PHP files in contributed or
   # custom modules or to run another PHP application in the same directory.
