Index: src/Plugin/Sitemap/Vocabulary.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Plugin/Sitemap/Vocabulary.php b/src/Plugin/Sitemap/Vocabulary.php
--- a/src/Plugin/Sitemap/Vocabulary.php	
+++ b/src/Plugin/Sitemap/Vocabulary.php	(date 1647441792838)
@@ -262,13 +262,13 @@
       '#items' => $list,
     ];
 
-    return [
+    return ($list) ? [
       '#theme' => 'sitemap_item',
       '#title' => $this->settings['title'],
       '#content' => $content,
       '#sitemap' => $this,
       // @todo Does a vocabulary cache tag exist?
-    ];
+    ] : [];
   }
 
   /**
